### ISSUE TITLE
Embeds are not showing in Guilds

### ISSUE THREAD

**Issue by Slimig** [iLinked1337/Anarchy#21]
The EmbedBuilder is working fine, but embeds do only work in dms for me. Is that a Bug?

**Comment by KibbeWater** [comment]
No, this is caused cause the guild you tried in haven't given you permission to send embeds

**Comment by iLinked1337** [comment]
I agree with @KibbeWater that this is probably a permission issue, tho i'm not sure what permission controls it.

**Comment by ColdYT** [comment]
embeded messages can be disabled 
https://gyazo.com/ed0e3a1a2dfbe53b0138f8181b4ce132
Thats why you wont be able to send embeded messages in some guilds
